changes

Aayam1o1 opened the pull request.

Aayam1o1 committed:
changes

NiteshTuladhar commented on parking/templates/index.html:
```diff
@@ -37,7 +37,7 @@
         <table class="table" id="myTable">
             <thead>
                 <tr>
-                    <th scope="col">Car Number</th>
```
**Changes to be made:**

1. Use snake cases for variable names and function names and pascal casing for classes.
2. Remove print statements before pushing your code to github.
3. You can use single form for create and update actions if possible.
4. Try to avoid same naming for project and apps.
5. I think it would be better if you create a new model for handling car parking information. By doing this you can add some more fields related to car parking like parking timestamps (when was the car parked , what time did it leave )

Aayam1o1 committed:
feat: Completed CRUD for parking wing with function based.

Completed listing of car details for car details through class based

Added feature to filter the car through date

Aayam1o1 committed:
Completed View more details for parking

Added new button edit to view the details for the vehicle details using DetailView

Added context in CarDetailView for edit of parking details

Aayam1o1 committed:
fix: changed variable name for optimal naming convention

NiteshTuladhar reviewed (approved):
Create a .gitignore file
